src/datacite_doi_helper_object.py

```diff
@@ -187,15 +187,42 @@ def create_dataset_draft_doi(self, dataset: dict, check_publication_status=True)
         else:
             raise KeyError('Either the entity_type of the given Dataset is missing or the entity is not a Dataset')
 
+    def create_collection_draft_doi(self, collection: dict) -> object:
```

Comment:
Let's add the missing comments to this function and distinguish it from the `create_dataset_draft_doi()`